### PR TITLE
add Japanese Question Answering Dataset (JaQuAD) dataset

### PR DESCRIPTION
**Pull request by xu3kev**
## Checklist for adding MMTEB dataset

Reason for dataset addition: Add [Japanese Question Answering Dataset (JaQuAD)](https://arxiv.org/abs/2202.01764) for Japanese language (not yet included in mteb)

- [X] I have tested that the dataset runs with the `mteb` package.
- [X] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [X] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [X] `intfloat/multilingual-e5-small`
- [X] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [X] I have considered the size of the dataset and reduced it if it is too big (2048 examples is typically large enough for most tasks)
- [X] Run tests locally to make sure nothing is broken using `make test`. 
- [X] Run the formatter to format the code using `make lint`. 
- [X] I have added points for my submission to the [POINTS.md](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) file.

### CODE REVIEW

**Review by KennethEnevoldsen — COMMENTED** [review]
Thanks for the contribution! Everything seems correct, so will just let the tests run

File: mteb/tasks/Retrieval/jpn/JaQuADRetrieval.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        bibtex_citation="""@misc{so2022jaquad,
+    title={{JaQuAD: Japanese Question Answering Dataset for Machine Reading Comprehension}},
+    author={ByungHoon So and Kyuhong Byun and Kyungwon Kang and Seongjin Cho},
+    year={2022},
+    eprint={2202.01764},
+    archivePrefix={arXiv},
+    primaryClass={cs.CL}
+}""",
+        n_samples=None,
+        avg_character_length=None,
```
[the remark this answers is not in the document]
Comment (xu3kev) [reply mteb/tasks/Retrieval/jpn/JaQuADRetrieval.py:40]: Thanks! Should the stats here about the one after subsampling to 2048 (as the guideline suggested) or the original dataset?

File: docs/mmteb/points.md
[the remark this answers is not in the document]
Comment (xu3kev) [reply docs/mmteb/points.md:1]: added!

File: mteb/tasks/Retrieval/jpn/JaQuADRetrieval.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (xu3kev) [reply mteb/tasks/Retrieval/jpn/JaQuADRetrieval.py]: looking at some other tasks in mteb, I added the stats for the original dataset to be consistent. Please let me know if the other way is preferred. Thanks!

[the remark this answers is not in the document]
Comment (KennethEnevoldsen) [reply mteb/tasks/Retrieval/jpn/JaQuADRetrieval.py]: After subsampling. Some of the older datasets have poor metadata, but we are trying to improve that.

[the remark this answers is not in the document]
Comment (KennethEnevoldsen) [reply mteb/tasks/Retrieval/jpn/JaQuADRetrieval.py]: (I believe most datasets on MMTEB aren't downsampled.)

[the remark this answers is not in the document]
Comment (xu3kev) [reply mteb/tasks/Retrieval/jpn/JaQuADRetrieval.py]: Fixed. Thank you!

**Review by KennethEnevoldsen — APPROVED** [review]

**Comment by KennethEnevoldsen** [comment]
I have set it to automerge once the tests pass. Thanks again for the contribution!

**Pull request merged**